runtime/common/src/purchase.rs

```diff
@@ -143,45 +163,44 @@ decl_error! {
 		/// Vesting schedule already exists for this account.
 		VestingScheduleExists,
 	}
-}
 
-decl_storage! {
-	trait Store for Module<T: Config> as Purchase {
-		// A map of all participants in the DOT purchase process.
-		Accounts: map hasher(blake2_128_concat) T::AccountId => AccountStatus<BalanceOf<T>>;
-		// The account that will be used to payout participants of the DOT purchase process.
-		PaymentAccount: T::AccountId;
-		// The statement purchasers will need to sign to participate.
-		Statement: Vec<u8>;
-		// The block where all locked dots will unlock.
-		UnlockBlock: T::BlockNumber;
-	}
-}
+	// A map of all participants in the DOT purchase process.
+	#[pallet::storage]
+	pub(super) type Accounts<T: Config> = StorageMap<
+		_,
+		Blake2_128Concat, T::AccountId,
```

Comment:
```suggestion
		Blake2_128Concat,
		T::AccountId,
```

Reply:
imho hasher and key type in same line makes `StorageMap`/`StorageDoubleMap` more readable: it's easier to tell which is the key part and the value part.